projects/Mallard/src/hooks/use-config-provider.tsx

```diff
@@ -17,6 +18,8 @@ interface ConfigState {
     }
     notificationsEnabled: boolean
     setNotifications: (setting: boolean) => Promise<void>
+    isSSR: boolean
```

Comment:
I would prefer if it referred to `isAppsRendering` as that is more descriptive.

Reply:
Absolutely - all updated!